docs/LLS_Metallicities-Wotta/input_files/mcmc.QSO1234+5678_z0.123456.in

```diff
@@ -0,0 +1,7 @@
+name                                            zabs      ezabs         ion     logn    elogn  flag              sample
```

Comment:
I'm ok with these being in pyigm, but they should
be in the pyigm.data folder somewhere.